torch/_functorch/partitioners.py

```diff
@@ -1411,7 +1746,15 @@ def classify_nodes(joint_module):
             for user in node.users:
                 node.dist_from_bw = min(node.dist_from_bw, user.dist_from_bw + 1)
 
-    saved_values = choose_saved_values_set(joint_graph, node_info, memory_budget=1)
+    memory_budget = config.activation_memory_budget
+    for node in joint_graph.nodes:
+        if isinstance(node.meta.get("memory_budget", None), float):
+            memory_budget = node.meta["memory_budget"]
```

Comment:
Do you plan to expose a way of making this visible from another part of the stack?

Reply:
Yes, I plan on adding a frontend API for it in `torch.utils.checkpoint`.

One thing we don't support right now, yet, and is a bit of nontrivial work to support, is supporting compiling a *subgraph* which has a memory budget defined on it.